server/tests/database/test_notification_job_data_handler.py

```
@@ -0,0 +1,221 @@
+
+    result = NotificationJobDataHandler.update_job(id, updated)
+    assert result is True
+
+    jobs = NotificationJobDataHandler.get_jobs_by_target(ORG_ID, 2, "decision_due")
+    assert len(jobs) == 1
+    assert jobs[0].targetId == 2
+    assert jobs[0].type == "decision_due"
+    assert jobs[0].payload == '{"decisionId": 2}'
+
+
+def test_update_job_not_found_returns_true(app):
```

Ruff (0.15.20)

[warning] Unused function argument: `app`

(ARG001)